desktop/windows/install.md

```diff
@@ -126,20 +126,20 @@ Log out and log back in for the changes to take effect.
 
 After downloading **Docker Desktop Installer.exe**, run the following command in a terminal to install Docker Desktop:
 
-```
-"Docker Desktop Installer.exe" install
+```console
+C:\> "Docker Desktop Installer.exe" install
 ```
 
 If you’re using PowerShell you should run it as:
 
-```
-Start-Process '.\win\build\Docker Desktop Installer.exe' -Wait install
+```console
+PS> Start-Process '.\win\build\Docker Desktop Installer.exe' -Wait install
```

Comment:
Is the `.\win\build\` actually correct here? I assume a user would download it to their current directory, not a `win/build` subdirectory 🤔

Reply:
Also slightly confused, as there seem to be 3 examples and both the first and last seem to be for `cmd.exe`, or is the third (`start /w ....`) for a different scenario?

Reply:
Oh good catch, no we should remove the `.\win\build\` prefix!

Reply:
As for the use of `start` for when using the Windows Command shell, it’s because the installer is actually a GUI application (as opposed to a Console application, see e.g. https://stackoverflow.com/a/7613999/1195832).
It’s a super annoying trait of Windows applications, and here it means that once invoked the installer process gets detached and the shell goes back to the prompt right away, but the logs are still written to the shell console…
`start` is a way to get around that (`/w` means «wait for the application to exit»)…